etc/async-aurora/src/lib.rs

```diff
@@ -0,0 +1,98 @@
+mod engine_types;
+use crate::engine_types::*;
+mod async_promise;
+use crate::async_promise::*;
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
+use near_sdk::{assert_self, env, ext_contract, near_bindgen, Gas, Promise};
+
+pub const GAS_RESERVED_FOR_CURRENT_CALL: Gas = 20_000_000_000_000;
```

Comment:
Suggesting changing the name. Also, are you sure 20 TGas will be enough? What's the real usage value?
```suggestion
pub const GAS_RESERVED_FOR_SUBMIT: Gas = 20_000_000_000_000;
```

Reply:
Fixed at bfe58440b083f686a714d95d5c4a968967c2bcdc